src/Lykke.Service.IndicesFacade/Services/IndicesFacadeService.cs

```diff
@@ -89,9 +99,30 @@ public async Task Handle(IndexTickPrice indexTickPrice)
             }
         }
 
+        private void InitializeAssetsDisplayIds()
+        {
+            try
+            {
+                var allAssets = _assetsService.AssetGetAll();
+                foreach (var assetId in _assetIdsClients.Keys.ToList())
+                {
+                    var asset = allAssets.Single(x => x.Id == assetId);
```

Comment:
please change to SingleOrDefault and if asset not exist in assetservice, then write warning and use IndexName

Reply:
fixed